src/Sulu/Bundle/MediaBundle/Api/Collection.php

```diff
@@ -40,19 +41,73 @@
  *          parameters = { "collection" = "expr(object.getId())", "limit" = 9999, "types" = "{types}" }
  *      )
  * )
+ * @Relation(
+ *      "self",
+ *      href = @Route(
+ *          "get_collection",
+ *          parameters = { "id" = "expr(object.getId())" }
+ *      )
+ * )
+ * @Relation(
+ *      "admin",
+ *      href ="expr('media/collections/edit:'~object.getId()~'/files')"
```

Comment:
Do you really need that? Couples the backend to the frontend, should be solved on the frontend side IMHO. @3rfan is already building something to "generate" routes on the frontend, right?